_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -122,10 +122,61 @@ This `iframe` element contains a link that is not part of its [sequential focus
 <iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 5
+
+This `iframe` element is [inert][] because of its own `inert` [attribute value][].
+
+```html
+<iframe inert srcdoc="<a href='/'>Home</a>"></iframe>
```

Comment:
I think the `iframe` shuld have a negative `tabindex` under the rule of thumb "the Inapplicable example should look bad if it wasn't inapplicable". The reason is that in implemetation reports "Passed" and "Inapplicable" results are essentially equivalent, so to detect wrong Applicability in a tool, we need the tool to (wrongly) report "Failed" rather than to wrongly report "Passed".
```suggestion
<iframe tabindex="-1" inert srcdoc="<a href='/'>Home</a>"></iframe>
```